README.md

```diff
@@ -59,6 +59,39 @@ build.sh.)
 docker build --build-arg TOMCAT_EXTRAS=false -t thinkwhere/geoserver-docker .
 ```
 
+### Native JAI / JAI ImageIO
+
+Native JAI and JAI ImageIO are included in the final image by default. You can remove
+them from the final image by setting the `JAI_IMAGEIO` build-arg to `false` when building
+the image.
```

Comment:
Might add a line here to explain when you might not want to include native JAI / imageio.  "you can remove them from the image if ..."